_includes/aip-nav.html

```diff
@@ -1,28 +1,46 @@
-{% assign scope = page.aip.scope or page.aip_index.scope %}
 <nav id="aip-nav" class="docs-component-nav">
   <ul class="nav-list">
-    <li class="nav-item nav-item-header">AIPs by category</li>
-    <li class="nav-item{% if page.url == '/general' %} nav-item-active{% endif %}">
-      <a href="{{ '/general' | relative_url }}">General</a>
-    </li>
-    {% for area in page.areas %}
-    {% if page.aip_index.scope == area[0] %}
-      {% assign scope = area[0] %}
-    {% endif %}
-    <li class="nav-item{% if page.aip_index.scope == area[0] %} nav-item-active{% endif %}">
-      <a href="{{ area[0] | relative_url }}">{{ area[1] }}</a>
-    </li>
-    {% endfor %}
-
-    <li class="nav-item nav-item-header">
-      {% if scope %}{{ page.areas[scope] }}{% else %}General{% endif %} AIPs
-    </li>
-    {% for p in site.pages %}{% if p.aip and p.aip.scope == scope -%}
-    <li class="nav-item{% if p.url == page.url %} nav-item-active{% endif %}">
-      <a href="{{ p.url | relative_url }}">
-        <span class="aip-number">{{ p.aip.id }}</span> {{ p.title }}
-      </a>
-    </li>
-    {% endif %}{% endfor -%}
```

Comment:
This is the heart of the commit. Liquid logic that builds the navigation bar based on the `_data/nav.yml`.

This is all deployed on my fork if you want to see it in action without checking this out:
https://rhamiltonsf.github.io/aip/general